src/main/frontend/Admin.html

```diff
@@ -1,44 +1,299 @@
+            console.log(rowcount)
+        } catch (error) {
+            console.error('Error fetching paginated data:', error.message);
+        }
+    }
+
+    let haschanged = false;
+    let cellContent;
+
+    function processData(csv) {
+        const allTextLines = csv.split(/\r\n|\n/);
+        let lines = [];
+        while (allTextLines.length) {
+            let line = allTextLines.shift().split(',');
+            lines.push(line);
+        }
+
+        drawOutput(lines);
+    }
+
+    function drawOutput(lines) {
+        // Clear previous data
+        document.getElementById("output").innerHTML = "";
+
+        const table = document.createElement("table");
+        table.className = "FullTable";
+
+        for (let i = 0; i < lines.length - 1; i++) {
+            const row = table.insertRow(-1);
+            row.id = i.toString();
+            row.className = "fullcsv";
+
+            if (i === 0) {
+                // Handling click for the header row
+                row.insertCell(-1).innerHTML = '';
+                row.firstChild.id = "buttons";
+                row.onclick = function (event) {
+                    if (event.target.tagName === "TD" && event.target.id !== "buttons") {
+                        if (cellContent === event.target.innerHTML) {
+                            sortOrder = sortOrder === 'asc' ? 'desc' : 'asc';
+                        } else {
+                            cellContent = event.target.innerHTML;
+                            sortOrder = 'asc';
+                        }
+                        fetchData(1, pagesize, sortOrder);
+                    }
+                };
+            } else if (i <= pagesize) {
+                // Buttons for non-header rows
+                row.insertCell(-1).innerHTML =
+                    '<button onclick="editData(this)">Edit</button>' +
+                    '<button onclick="deleteData(this)">Delete</button>';
+            }
+            for (let j = 0; j < lines[i].length; j++) {
+                const cell = row.insertCell(-1);
+                cell.appendChild(document.createTextNode(lines[i][j]));
+            }
+        }
+        document.getElementById("output").appendChild(table);
+    }
+
+    async function deleteData(button) {
+        // Get the parent row of the clicked button
+        let row = button.parentNode.parentNode;
+        deleterow = row.children;
+        // Remove the row from the table
+        row.parentNode.removeChild(row);
+        haschanged = true;
+        let url = `http://127.0.0.1:5000/scene/complete/table?Deleterow=${deleterow[1].innerHTML}`;
+        await fetch(url);
+    }
+
+    function editData(button) {
+    haschanged = true;
+    let row = button.parentNode.parentNode;
+
+    // Create a form element
+    let form = document.createElement("form");
+    form.className = "editform"
+    let fields
+
+    // Define the fields you want to edit
+    if (row.cells[6].innerHTML === "EHAM")
```

Comment:
Is it maybe possible to GET these values from the database instead of harcoded values